sources/packages/web/src/components/common/sin/AddExpiryDate.vue

```diff
@@ -2,8 +2,9 @@
   <v-form ref="addExpiryDateForm">
     <modal-dialog-base
       title="Add expiry date"
-      :showDialog="showDialog"
-      max-width="730"
+      :dialog-type="DialogTypes.question"
```

Comment:
I believe that `dialog-type` is old and we no longer need it. The code associated with it can also be removed?
Is it causing any impact on the UI?

Reply:
`dialogType` is a required field in the ModalDialogBase component 
![image](https://user-images.githubusercontent.com/77353155/195955169-8342b001-e47b-4a77-8203-7f8ee17cc21f.png)
and because of that below error was there
![image](https://user-images.githubusercontent.com/77353155/195955222-16088550-d55a-4646-a997-3773a0672fd8.png). so shall i remove this property from the component

Reply:
wait is this error happing to only me

Reply:
property removed